### PR TITLE
luci-app-hysteria: add luci-app-hysteria

### PR DESCRIPTION
**Pull request by findlayfeng**
- [x] This PR is not from my *main* or *master* branch :poop:, but a *separate* branch :white_check_mark:
- [x] Each commit has a valid :black_nib: `Signed-off-by: <my@email.address>` row (via `git commit --signoff`)
- [x] Each commit and PR title has a valid :memo: `<package name>: title` first line subject for packages
- [x] Incremented :up: any `PKG_VERSION` in the Makefile
- [x] Tested on: (architecture, openwrt version, browser) :white_check_mark:
- [ ] \( Preferred ) Mention: @ the original code author for feedback
- [ ] \( Preferred ) Screenshot or mp4 of changes:
- [ ] \( Optional ) Closes: e.g. openwrt/luci#issue-number
- [x] \( Optional ) Depends on: e.g. openwrt/packages#24701 in sister repo
- [ ] Description: (describe the changes proposed in this PR)

### CODE REVIEW

**Comment by hnyman** [comment]
File example.pot is wrongly named. should be hysteria.pot

File: applications/luci-app-hysteria/htdocs/luci-static/resources/view/hysteria/client.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+		s.tab('tproxy', _('Tproxy'));
+		s.tab('dnsmasq', _('Dnsmasq'));
+
+		s.taboption('general', form.Flag, 'enabled',
+			_('Enabled'),
+			_('Enable Hysteria Client instance.'));
+
+		o = s.taboption('general', form.Value, 'confdir',
+			_('Configuration Directory'),
+			_('Directory where the Hysteria Client configuration file are located.')
```
Comment (systemcrash) [inline applications/luci-app-hysteria/htdocs/luci-static/resources/view/hysteria/client.js:27]: file is
files are

File: applications/luci-app-hysteria/htdocs/luci-static/resources/view/hysteria/client.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
+'use strict';
+'require view';
+'require form';
+
+return view.extend({
+	render: function() {
+		var m, s, o;
+
+		m = new form.Map('hysteria', _('Hysteria'),
+			_('A feature-packed proxy & relay tool optimized for lossy'));
```
Comment (systemcrash) [inline applications/luci-app-hysteria/htdocs/luci-static/resources/view/hysteria/client.js:10]: for lossy what?

File: applications/luci-app-hysteria/htdocs/luci-static/resources/view/hysteria/client.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+		o.render = function(option_index, section_id, in_table) {
+			this.placeholder = section_id;
+
+			return Promise.resolve(this.cfgvalue(section_id))
+				.then(this.renderWidget.bind(this, section_id, option_index))
+				.then(this.renderFrame.bind(this, section_id, in_table, option_index));
+		};
+
+		o = s.taboption('tproxy', form.Value, 'tproxy_set',
+			_('Nft Tabel Set')
```
Comment (systemcrash) [inline applications/luci-app-hysteria/htdocs/luci-static/resources/view/hysteria/client.js:86]: Table

Comment (systemcrash) [reply applications/luci-app-hysteria/htdocs/luci-static/resources/view/hysteria/client.js:86]: Nft => NFT

File: applications/luci-app-hysteria/htdocs/luci-static/resources/view/hysteria/server.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
+'use strict';
+'require view';
+'require form';
+
+return view.extend({
+	render: function() {
+		var m, s, o;
+
+		m = new form.Map('hysteria', _('Hysteria Server'),
+			_('A feature-packed proxy & relay tool optimized for lossy'));
```
Comment (systemcrash) [inline applications/luci-app-hysteria/htdocs/luci-static/resources/view/hysteria/server.js:10]: for lossy what?

File: applications/luci-app-hysteria/htdocs/luci-static/resources/view/hysteria/server.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+		s.addremove = true;
+		s.addbtntitle = _('Add server instance', 'Hysteria Serverd instance');
+
+		s.option(form.Flag, 'enabled',
+			_('Enabled'),
+			_('Enable Hysteria Server instance.'));
+
+		o = s.option(form.Value, 'confdir',
+			_('Configuration Directory'),
+			_('Directory where the Hysteria Server configuration file are located.')
```
Comment (systemcrash) [inline applications/luci-app-hysteria/htdocs/luci-static/resources/view/hysteria/server.js:23]: file is
files are

**Review by systemcrash — CHANGES_REQUESTED** [review]
mostly spelling/grammar

**Comment by findlayfeng** [comment]
@systemcrash Thank you for your review, the revision has been completed

**Review by systemcrash — APPROVED** [review]

**Pull request closed without merging**